tests/vmi_lifecycle_test.go

```diff
@@ -110,16 +111,39 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 			pod := pods.Items[0]
 			handlerNamespace := pod.GetNamespace()
-			err = virtClient.CoreV1().Pods(handlerNamespace).Delete(context.Background(), pod.Name, metav1.DeleteOptions{})
-			Expect(err).ToNot(HaveOccurred())
 
-			Eventually(func() string {
-				n, err := virtClient.CoreV1().Nodes().Get(context.Background(), pod.Spec.NodeName, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
+			By("setting up a watch on Nodes")
+			nodeWatch, err := virtClient.CoreV1().Nodes().Watch(context.Background(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			signalSchedulableNode := func(stopCn <-chan bool, eventsCn <-chan k8sWatch.Event, scheduleSignalCn chan<- bool) {
```

Comment:
Does it make sense to use gomega channel matchers for this instead of the for/case

Reply:
@qinqon I don't understand
for/case is used to catch events from the channels when it is run as goroutine
We already do the channel assertion at the end using gomega matcher